index.html

```diff
@@ -43,35 +43,36 @@
 
       <div class="row justify-content-center">
         <div class="col-xl-9 d-flex align-items-center justify-content-between">
-          <h1 class="logo"><a href="index.html">KnightOne</a></h1>
+          <h1 class="logo"><a href="index.html">ProfNITT</a></h1>

```

Comment:
do we need it to be "index.html", can we have it as a tag that scrolls to the top or something? Because, when we click on it, it reloads the page.